inst/rmarkdown/templates/ctex/skeleton/skeleton.Rmd

```diff
@@ -8,7 +8,6 @@ documentclass: ctexart
 keywords:
   - 中文
   - R Markdown
-documentclass: ctexart
```

Comment:
This line shouldn't be deleted. It's a document class based on the LaTeX package **ctex**.

Reply:
But it is already line 7 of the same file ? 
https://github.com/rstudio/rticles/pull/323/files#diff-df04ce7c0c57ebe73c4c44be67b01f45L7

So I get a duplicated field warning from yaml reading.

Reply:
Oops. I didn't see that. You are right. We should delete the duplicate.